requirements/pip.txt

```diff
@@ -0,0 +1,18 @@
+cloudpickle
+pygments>=2.0
+qtconsole>=4.2.0
+nbconvert
+sphinx
+pylint
+psutil
+qtawesome>=0.5.0
+qtpy>=1.2.0
+pickleshare
+pyzmq
+chardet>=2.0.0
+numpydoc
+pyqt5>=5.6
+keyring
+spyder-kernels>=1.0
+python-language-server==0.19.0
+qdarkstyle>=2.6.3
```

Comment:
According to @dpizetta we should require 2.6.4 since it fixed some important bugs that broke QDarkStyle for Spyder users running under Python 2 .

Reply:
Ok, will do in this PR.